lms/templates/navigation.html

```diff
@@ -180,6 +159,34 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
       </%block>
     </ol>
     % endif
+    % if static.show_language_selector():
```

Comment:
Is this code block a duplicate of what is present in header.html?  If so is there any way we can combine the two into a reusable include or something?